deploy/secrets.yaml

```diff
@@ -1,19 +1,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: my-cluster-name-secrets
+  name: cluster1-secrets
 type: Opaque
-stringData:
-  MONGODB_BACKUP_USER: backup
-  MONGODB_BACKUP_PASSWORD: backup123456
-  MONGODB_DATABASE_ADMIN_USER: databaseAdmin
-  MONGODB_DATABASE_ADMIN_PASSWORD: databaseAdmin123456
-  MONGODB_CLUSTER_ADMIN_USER: clusterAdmin
-  MONGODB_CLUSTER_ADMIN_PASSWORD: clusterAdmin123456
-  MONGODB_CLUSTER_MONITOR_USER: clusterMonitor
-  MONGODB_CLUSTER_MONITOR_PASSWORD: clusterMonitor123456
-  MONGODB_USER_ADMIN_USER: userAdmin
-  MONGODB_USER_ADMIN_PASSWORD: userAdmin123456
-  PMM_SERVER_API_KEY: apikey
-  #PMM_SERVER_USER: admin
-  #PMM_SERVER_PASSWORD: admin
+data:
```

Comment:
we need to have all these keys to show example for users

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1112/commits/e8f6ce8911e7c1deb5c4e53815b85a2f56da7c98